src/qililab/slurm.py

```diff
@@ -85,13 +86,17 @@ def submit_job(line: str, cell: str, local_ns: dict) -> None:
     args = parse_argstring(submit_job, line)
     output = args.output
     partition = args.partition
+    gres = args.gres
     job_name = args.name
     time_limit = int(args.time)
     folder_path = args.logs
     execution_env = args.execution_environment
     begin_time = args.begin
     low_priority = args.low_priority
 
+    if gres is None:
+        raise ValueError("GRES needs to be provided!")
```

Comment:
Would be nice to return an error with a list of valid options.
Do we have the list of GRES chips?

If we don't have the list, at least an example? Like `--gres aQPU1`
Otherwise, some clue on when to find it?

Reply:
yes, this was to be included in the documentation! But I will return the values in the error, good idea.

Reply:
see the update code: 
![image](https://github.com/user-attachments/assets/81885421-d744-4f59-ac91-da50ab5d3ef2)

Reply:
see the updated code: 
![image](https://github.com/user-attachments/assets/04ef9220-4d7c-4d52-95b3-8a4cb4a756d6)